etc/kaleidoscope-builder.conf

```diff
@@ -150,6 +151,12 @@ if [ "${uname_S}" = "Darwin" ]; then
         fi
     }
 
+    if [ "${ARCH}" = "virtual" ]; then
```

Comment:
If this doesn't actually work, I'd probably not merge it in for now, since it's typing to a specific build of the gnu tools

Reply:
Fixed, thanks!